### ISSUE TITLE
Sup huhuang03

### ISSUE THREAD

**Issue by brunubarbosa** [huhuang03/huhuang03.github.io#1]
Hey dude, I'd like to know if it's you the owner of this chrome extension https://chrome.google.com/webstore/detail/ytb-not-interested-button/okfiigkfppbpfjaiffmllkkfbpdnhofn
If it's you, what you do think about turning its code into an open-source code?
Sorry about this approach I didn't find any other way to talk to you

**Comment by huhuang03** [comment]
Yes, it's already open souced. https://github.com/huhuang03/ytb_dni
